packages/server/src/instances/service.ts

```diff
@@ -110,12 +110,13 @@ export class InstanceService extends Service {
   }
 
   async destroy() {
-    const keys = this.cache.keys()
+    if (!this.cache) {
+      return
+    }
```

Comment:
Note that it's possible for the server to crash in the constructor so cache can be undefined in that scenario